zhaquirks/tuya/ts0601_trv_sas.py

```diff
@@ -160,23 +128,21 @@ def map_attribute(self, attribute, value):
 
         if attribute == "occupied_heating_setpoint":
             # centidegree to decidegree
-            return {OCCUPIED_HEATING_SETPOINT_COMMAND_ID: round(value / 10)}
+            return {SASSWEL_HEATING_SETPOINT_ATTR: round(value / 10)}
```

Comment:
I feel like there is something wrong here.

At first sign there is an asymetric behavior with the temp attributes (`local_temperature` and `occupied_heating_setpoint`).
The `_update_attribute` is updating both with `value * 10`.
And then here in `map_attribute` is only updating the `occupied_heating_setpoint `(value/10)`.
Are you able to check if this part is fine?

Reply:
There is no need to handle local temperature in `map_attribute` because this value is a read only property of the device. Only the heating setpoint can be changed by the user (HA dashboard) and only this value requires translation from ZCL to Tuya format.